src/main/java/dev/snowdrop/release/App.java

```diff
@@ -170,10 +183,20 @@ public void startRelease(
     
     @CommandLine.Command(name = "list-cves", description = "List CVEs for the specified release or, if not specified, unresolved CVEs")
     public void listCVEs(
+        @CommandLine.Option(names = {"-r", "--release"}, description = "Release the CVE list to github") boolean release,
+        @CommandLine.Option(names = {"-o", "--token"}, description = "Github API token. Required if --release is enabled") String token,
         @CommandLine.Parameters(description = "Release for which to retrieve the CVEs, e.g. 2.2.10", arity = "0..1") String version
     ) throws Throwable {
         final var cves = cveService.listCVEs(Optional.ofNullable(version));
         reportStatus(cves);
+        String mdReport = cveReportSvc.buildMdReport(cves, "Version - " + Optional.ofNullable(version).orElse(" ALL OPEN"));
```

Comment:
This should only be done if we're releasing, no point in generating the MD otherwise.

Reply:
I thought the idea was to have it published so it could be easily accessed. Shouldn't it be independent? With this option it can be launched optionally.

Reply:
I meant "releasing" as in publishing the CVEs to Github. The markdown doesn't need generated if the "release" option is not activated. Even more, it only needs to be generated if both release and token options are present.

Reply:
I see, good point.

Reply:
fixed